clientgui/wizardex.h

```diff
@@ -246,7 +246,7 @@ class WXDLLIMPEXP_ADV wxWizardEx : public wxWizardBaseEx
     wxStaticBitmap *m_statbmp;  // the control for the bitmap
 
     // Whether user called SetBorder()
-    bool m_calledSetBorder;
+//    bool m_calledSetBorder;
```

Comment:
I believe we can just remove this dead code

Reply:
This code was not written by anyone at BOINC. It was taken by Rom Walton from wxWidgets; see the copyright at the top of the file. So I am reluctant to change it beyond adding the `//`.

Reply:
Ok, one day I'll go through all the commented code and remove it (especially all the code that was commented decades ago ^^)